src/model/fxQuotes.js

```diff
@@ -49,6 +49,7 @@ class FxQuotesModel {
   async validateFxQuoteRequest (fspiopSource, fspiopDestination, fxQuoteRequest) {
     await this.db.getParticipant(fspiopSource, LOCAL_ENUM.PAYER_DFSP, fxQuoteRequest.conversionTerms.sourceAmount.currency, ENUM.Accounts.LedgerAccountType.POSITION)
     await this.db.getParticipant(fspiopDestination, LOCAL_ENUM.PAYEE_DFSP, fxQuoteRequest.conversionTerms.sourceAmount.currency, ENUM.Accounts.LedgerAccountType.POSITION)
+    // Should we be validating participant against the targetCurrency too?
```

Comment:
It seems above logic is not correct. There are few issues
- I think the participantTypes should be 'Initiating FSP' and 'Counterparty FSP' instead of PAYER_DFSP and PAYEE_DFSP
- We can validate counterPartyFsp with both currencies sourceCurrency and targetCurrency.
- initiatingFsp validation depends on the use case and I think we can ignore that for now.